params_test.go

```diff
@@ -3813,3 +3813,18 @@ func TestReplaceSidecarTagsWithDigest(t *testing.T) {
 		assert.True(t, strings.HasPrefix(params.Sidecars[1].Image, "estafette/estafette-docker-cache-heater@sha256:"))
 	})
 }
+
+func TestVisibilityPublicDeprecationWarning(t *testing.T) {
+
+	t.Run("ReplacesFirstSidecarImageTagWithDigest", func(t *testing.T) {
```

Comment:
Can you name this test appropriately to cover what it tests?

Reply:
And it can be moved to TestValidateRequiredProperties; the inner name is about what it tests, the outer function is for which function is under test.

Reply:
fixed